docs/source/general/setup.rst

```diff
@@ -2,6 +2,25 @@
 Setup
 =====
 
+Create a virtual environment
+----------------------------
+
+Please use a `python virtual environment <https://docs.python.org/3/library/venv.html>`_. It keeps everybody on the same page, helps reproducing bugs
+and resolving problems. Otherwise, you may encounter errors.
+
+We suggest you to use `virtualenvwrapper <https://virtualenvwrapper.readthedocs.io>`_ for this task.
+
+.. code-block:: shell
+
+    sudo pip3 install virtualenvwrapper    
+    source virtualenvwrapper.sh #check note if you have an error
```

Comment:
please just link the virtualenvwrapper installation page and remove the previous 2 lines, never ever duplicate information that then will need to be maintained twice. You can leave here the subsequent line